src/main/java/edu/cornell/kfs/fp/batch/xml/cloudcheckr/GroupLevel.java

```diff
@@ -0,0 +1,144 @@
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String groupValue;
+    
+    @XmlElement(name = "FriendlyName", namespace = StringUtils.EMPTY)
+    @XmlJavaTypeAdapter(NormalizedStringAdapter.class)
+    private String friendlyName;
+    
+    @XmlElement(name = "Cost", namespace = StringUtils.EMPTY)
+    @XmlJavaTypeAdapter(KualiDecimalXmlAdapter.class)
+    private KualiDecimal cost;
+    
+    @XmlElement(name = "UsageQuantity", namespace = StringUtils.EMPTY)
+    protected Double usageQuantity;
+    
+    @XmlElementWrapper(name = "NextLevel", namespace = StringUtils.EMPTY)
+    @XmlElement(name = "GroupingLevel", namespace = StringUtils.EMPTY)
+    private List<GroupLevel> nextLevel;
+    
+    public GroupLevel() {
+        nextLevel = new ArrayList<GroupLevel>();
+        cost = KualiDecimal.ZERO;
+        usageQuantity = new Double(0);
+    }
+    
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("groupName: ").append(groupName);
+        sb.append(" groupValue: ").append(groupValue);
+        sb.append(" friendlyName: ").append(friendlyName);
+        sb.append(" cost: ").append(cost);
+        sb.append(" usageQuantity: ").append(usageQuantity);
+        nextLevel.stream().forEach(level -> sb.append(" nextlevel: ").append(level.toString()).append(", "));
+        return sb.toString();
+    }
+    
+    public boolean isAwsAccountGroupLevel() {
+        return StringUtils.equals(groupName, CuFPConstants.CLOUDCHECKR.GROUP_LEVEL_AWS_ACCOUNT);
+    }
+    
+    public boolean isCostCenterGroupLevel() {
+        return StringUtils.equals(groupName, CuFPConstants.CLOUDCHECKR.GROUP_LEVEL_COST_CENTER);
+    }
+
+    public String getGroupName() {
+        return groupName;
+    }
+
+    public void setGroupName(String groupName) {
+        this.groupName = groupName;
+    }
+
+    public String getGroupValue() {
+        return groupValue;
+    }
+
+    public void setGroupValue(String groupValue) {
+        this.groupValue = groupValue;
+    }
+
+    public String getFriendlyName() {
+        return friendlyName;
+    }
+
+    public void setFriendlyName(String friendlyName) {
+        this.friendlyName = friendlyName;
+    }
+
+    public KualiDecimal getCost() {
+        return cost;
+    }
+
+    public void setCost(KualiDecimal cost) {
+        this.cost = cost;
+    }
+
+    public Double getUsageQuantity() {
+        return usageQuantity;
+    }
+
+    public void setUsageQuantity(Double usageQuantity) {
+        this.usageQuantity = usageQuantity;
+    }
+
+    public List<GroupLevel> getNextLevel() {
+        return nextLevel;
+    }
+
+    public void setNextLevel(List<GroupLevel> nextLevel) {
+        this.nextLevel = nextLevel;
+    }
+    
+    @Override
+    public boolean equals(Object o) {
+        boolean equals = false;
+        if (ObjectUtils.isNotNull(o) && o instanceof GroupLevel) {
+            GroupLevel otherGl = (GroupLevel) o;
```

Comment:
otherGl should be changed to otherGroupLevel

Reply:
sure